db/migrate/20200408173214_add_foreign_key_constraints.rb

```diff
@@ -12,7 +12,8 @@ def change
     # add_foreign_key :activities, :organisations, column: "reporting_organisation_id"
 
     add_foreign_key :transactions, :activities, on_delete: :cascade
-    add_foreign_key :budgets, :activities, on_delete: :cascade
+    # Covered in 20200407140854
+    # add_foreign_key :budgets, :activities, on_delete: :cascade
```

Comment:
This has already been applied on all environments so we probably shouldn't retrospectively edit it as this migration will never be rerun. We should assume it's already there and in our new migration approach it as a change.

Reply:
Yes I wasn't sure about this! I will remove it